docs-md/developer-guide/create-a-table.md

```diff
@@ -250,30 +282,39 @@ SELECT * FROM pageviews_table EMIT CHANGES;
-1557183929786 | 1557183929786|+|User_5|+|Page_12 : Window{start=1557183900000 end=-} | 1557183929786 | User_5 | Page_12 | 1
-1557183931095 | 1557183931095|+|User_3|+|Page_43 : Window{start=1557183900000 end=-} | 1557183931095 | User_3 | Page_43 | 1
-1557183930184 | 1557183930184|+|User_1|+|Page_29 : Window{start=1557183900000 end=-} | 1557183930184 | User_1 | Page_29 | 1
-1557183930727 | 1557183930726|+|User_6|+|Page_93 : Window{start=1557183900000 end=-} | 1557183930726 | User_6 | Page_93 | 1
++---------------+---------------+---------------+------------------+--------+---------+------+
+| ROWTIME       | WINDOWSTART   | WINDOWEND     | ROWKEY           | USERID | PAGEID  | TOTAL|
++---------------+---------------+---------------+------------------+--------+---------+------+
+| 1557183919786 | 1557183900000 | 1557183960000 | User_5|+|Page_12 | User_5 | Page_12 | 1    |
+| 1557183929488 | 1557183900000 | 1557183960000 | User_9|+|Page_39 | User_9 | Page_39 | 1    |
+| 1557183930211 | 1557183900000 | 1557183960000 | User_1|+|Page_79 | User_1 | Page_79 | 1    |
+| 1557183930687 | 1557183900000 | 1557183960000 | User_9|+|Page_34 | User_9 | Page_34 | 1    |
+| 1557183929786 | 1557183900000 | 1557183960000 | User_5|+|Page_12 | User_5 | Page_12 | 2    |
+| 1557183931095 | 1557183900000 | 1557183960000 | User_3|+|Page_43 | User_3 | Page_43 | 1    |
+| 1557183930184 | 1557183900000 | 1557183960000 | User_1|+|Page_29 | User_1 | Page_29 | 1    |
+| 1557183930727 | 1557183900000 | 1557183960000 | User_6|+|Page_93 | User_6 | Page_93 | 3    |
 ^CQuery terminated
 ```
 
+!!! note
+        It is possible for the same key to be output multiple time when emitting changes
+        to the table. This is because each time the row in the table changes it will be emitted.
+
 Look up the value for a specific key within the table by using a SELECT
 statement.
 
 ```sql
-SELECT * FROM pageviews_table WHERE ROWKEY='1557183929488|+|User_9|+|Page_39';
+SELECT * FROM pageviews_table WHERE ROWKEY='User_9|+|Page_39';
 ```
 
 Your output should resemble:
 
 ```
- ROWKEY STRING KEY                | WINDOWSTART BIGINT KEY | VIEWTIME BIGINT | USERID STRING | PAGEID STRING | TOTAL BIGINT
-----------------------------------------------------------------------------------------------------------------------------
- 1557183929488|+|User_9|+|Page_39 | 1557183900000          | 1557183929488   | User_9        | Page_39       | 1
-----------------------------------------------------------------------------------------------------------------------------
++-----------------+---------------+---------------+--------+---------+-------+
+ ROWKEY           | WINDOWSTART   | ROWTIME       | USERID | PAGEID  | TOTAL |
+------------------+---------------+---------------+--------+---------+-------+
+ User_9|+|Page_39 | 1557183900000 | 1557183929488 | User_9 | Page_39 | 1     |
+Query terminated
```

Comment:
Update to:
- new formatting
- reflect removal of `viewtime` from composite key

Note: its intentional that `WINDOWEND` is not included. This functionality is not there yet.